mkdocs.yml

```diff
@@ -203,8 +203,9 @@ nav:
         - Spanish Localization Bundle: developer-guide/etendo-classic/bundles/spanish-localization-bundle.md
         - Etendo BI Bundle: developer-guide/etendo-classic/bundles/etendo-bi-bundle.md
     - Etendo RX:
-      - Getting Started: developer-guide/etendo-rx/getting-started.md  
-      - First Steps: developer-guide/etendo-rx/first-steps.md
+      - Getting Started: developer-guide/etendo-rx/getting-started.md 
+      - Tutorial:  
+        - First Steps: developer-guide/etendo-rx/first-steps.md
```

Comment:
You should create a "tutorials" folder and move the file into it to respect the navigation structure.